simulation/amaru-sim/src/simulator/mod.rs

```diff
@@ -151,7 +151,7 @@ async fn run_simulator(
                 let events = match mk_message(msg, span) {
                     Ok(event) => match event {
                         PullEvent::RollForward(peer, point, raw_header, _span) => {
-                            info!("got forward event {:?}", &point);
+                            info!("got forward event {:?}", &point.pretty_point());
```

Comment:
Those kind of `info` do not quite align with the `tracing` philosophy. Is it required by the simulator @stevana ?

Reply:
I will remove it, it's useless anyway